dev/MRTCore/build/versions.props

```diff
@@ -21,8 +21,8 @@
     <CurrentWilPackageVersion>1.0.190716.2</CurrentWilPackageVersion>
     <MuxCustomBuildTasksPackageVersion>1.0.83-winui3</MuxCustomBuildTasksPackageVersion>
     <MicrosoftCsWinRTPackageVersion>0.1.0-prerelease.200629.3</MicrosoftCsWinRTPackageVersion>
```

Comment:
Can this be removed to avoid possible confusion? It doesn't seem to be used anymore.